test/python/circuit/test_extensions_standard.py

```diff
@@ -1486,5 +1488,52 @@ def test_kwarg_deprecation(self, instr_name, inst_class, n_params, kwarg_map):
         self.assertEqual(cargs, [])
 
 
+@ddt
+class TestStandardEquivalenceLibrary(QiskitTestCase):
```

Comment:
can you move this test to `test/python/circuit/test_gate_definitions.py`? That file was created to test standard equivalence rules are correct.

 I think the ddt way you are doing here is nicer so probably can just overwrite that. But this current `test_extensions_standard.py` file is kind of huge and a mess.